kyaml/yaml/example_test.go

```diff
@@ -893,3 +893,111 @@ func ExampleRNode_ElementValues() {
 	// Output:
 	// [foo bar baz] <nil>
 }
+
+var (
+	deployment RNode
+	configMap  RNode
+)
+
+func ExampleRNode_mutatePrimitiveField() {
```

Comment:
It's great to have these as part of the docs, but it would be even better if they had expected output both to be shown in the docs and so that these can be run as tests, like the rest of the examples in this file. Can you do that please?